integrated_channels/xapi/statements/learner_course_completion.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+"""
+X API Statement when learner completes a course.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from tincan import Activity, ActivityDefinition, Agent, Context, Extensions, LanguageMap, Result, Score, Verb
+
+from integrated_channels.xapi.statements.base import EnterpriseStatement
+
+
+class LearnerCourseCompletionStatement(EnterpriseStatement):
+    """
+    X-API Statement to serialize data related to course completion.
+    """
+
+    def __init__(self, user, course_overview, user_details, course_details, course_grade, *args, **kwargs):
+        """
+        Initialize and populate statement with learner info and course info.
+
+        Arguments:
+            user (User): Auth User object containing information about the learner enrolling in the course.
+            course_overview (CourseOverview): course overview object containing course details.
+            user_details (dict): A dict object containing learner info we want to send in X-API statement payload.
+            course_details (dict): A dict object containing course info we want to send in X-API statement payload.
+            course_grade (CourseGrade): User grade in the course.
+        """
+        kwargs.update(
+            actor=self.get_actor(user.username, user.email),
+            verb=self.get_verb(),
+            object=self.get_object(course_overview.display_name, course_overview.short_description),
+            result=self.get_result(course_grade),
+            context=self.get_context(user_details, course_details)
+        )
+        super(LearnerCourseCompletionStatement, self).__init__(*args, **kwargs)
+
+    def get_actor(self, username, email):
+        """
+        Get actor for the statement.
+        """
+        return Agent(
+            name=username,
+            mbox='mailto:{email}'.format(email=email),
+        )
+
+    def get_verb(self):
+        """
+        Get verb for the statement.
+        """
+        return Verb(
+            id='http://adlnet.gov/expapi/verbs/completed',
+            display=LanguageMap({'en-US': 'completed'}),
```

Comment:
This is fine for now, but I do wonder if we'll need configurable locale as we just added to SF. We can cross that bridge when it comes